Code/Framework/AzFramework/AzFramework/Physics/Common/PhysicsSceneQueries.h

```diff
@@ -71,12 +71,12 @@ namespace AzPhysics
             //! Performance hint flag for ShapeCasts when it is known upfront there's no initial overlap.
             //! NOTE: using this flag may cause undefined results if shapes are initially overlapping.
             AssumeNoInitialOverlap = (1 << 4),
-            MeshMultiple = (1 << 5), //!< Report all hits for meshes rather than just the first. Not applicable to ShapeCast queries.
+            AnyHit = (1 << 5), //!< Report any first hit. Used for geometries that contain more than one primitive. For meshes,
+                               //!< if neither MeshMultiple nor AnyHit is specified, a single closest hit will be reported.
+            MeshMultiple = (1 << 6), //!< Report all hits for meshes rather than just the first. Not applicable to ShapeCast queries.
```

Comment:
question: As `MeshMultiple` has changed from `(1 << 5)` to `(1 << 6)`, is there any chance this could impact or break any serialized data? Do we need to think of any kind of upgrade/conversion path or is this relatively safe you think?

Reply:
These are not serialized anywhere. 
Obviously technically external code could keep the hit flags saved somewhere in the database or file, but that'd be outside of our reflection system.

Reply:
Okay understood thanks for clarifying 👍